codex-process-feasibility/README.md

```diff
@@ -0,0 +1,58 @@
+# NUM CODEX Feasibility Process
+
+The feasibility process is used to run a distributed feasibility query. A feasibility query contains multiple criteria and returns one population count per organization. The feasibility query is requested by one organization (usually the Zentrale Antrags- und Registerstelle (ZARS)) and executed in multiple organizations (usually Datenintegrationszentren (DIZ)).
```

Comment:
Always try to go for active voice instead of passive voice.

**Why?:**
- Active voice tends to be shorter and more precise.
- Passive voice obfuscates ideas and turns sentences on their head (mental load).
- Readers typically translate passive to active voice in their head.

**How?:**
Avoid sentence structures in the form of `form of be + past participle verb`.

**Example:**
```suggestion
A distributed feasibility query runs based on the feasibility process. A feasibility query contains multiple criteria and returns one population count per organization. An organization (usually the **Zentrale Antrags- und Registerstelle (ZARS)**) requests a feasibility query to start off the process. Once started multiple organizations (usually **Datenintegrationszentren (DIZ)**) execute this query and report results back.
```